src/shared/utils/logger.go

```diff
@@ -0,0 +1,41 @@
+package utils
+
+import (
+	"log"
+	"os"
+)
+
+// file path of our logs
+var path = "logs/logs.txt"
+
+//setting up logs type
+func Log(txt string, typ int) {
+	if err := ensureDir("logs"); err != nil {
+		os.Exit(1)
+	}
+	file, err := os.OpenFile(path, os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0666)
+	if err != nil {
+		log.Fatal(err)
+	}
+	switch typ {
+	case 0:
+		log.New(file, "INFO: ", log.Ldate|log.Ltime|log.Lshortfile).Print(txt)
+	case 1:
+		log.New(file, "WARNING: ", log.Ldate|log.Ltime|log.Lshortfile).Print(txt)
+	default:
+		log.New(file, "ERROR: ", log.Ldate|log.Ltime|log.Lshortfile).Print(txt)
```

Comment:
`log.New` returns pointers to unique log objects, we should reuse those objects instead of wasting memory after each log.

Reply:
ayay captain